### PR TITLE
Use native YAML

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
cloud/amazon/route53

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @minichate please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/amazon/route53_zone.py [hunk 1]
```diff
@@ -54,25 +54,34 @@
...
+    vpc_id: '{{ vpc.vpc_id }}'
+    vpc_region: '{{ ec2_region }}'
+    zone: '{{ vpc_dns_zone }}'
     state: present
     register: zone_out
 
-- debug: var=zone_out
-
+- debug:
+    var: zone_out
```
Comment (gundalow) [inline cloud/amazon/route53_zone.py:84]: I'd need to check, though I think this should have quotes and double curly braces.

**Review by gundalow — CHANGES_REQUESTED** [review]

File: cloud/amazon/route53_zone.py (hunk 1 above)
Comment (Fale) [reply cloud/amazon/route53_zone.py:84]: If you want to print a variable in "msg" yes, in "var" you should not. We can discuss on the "safety" of this feature. http://docs.ansible.com/ansible/debug_module.html

**Comment by gundalow** [comment]
@fale thanks, I was remembering `msg` v `var`
If there is any extra discussion needed that can be done in one of the public IRC meetings.

**Review by gundalow — APPROVED** [review]

**Pull request merged**